client/src/js/references/components/Detail/Targets/__test__/Targets.test.js

```diff
@@ -0,0 +1,80 @@
+import { mapStateToProps, mapDispatchToProps, Targets, StyledAddTargetsButton } from "../Targets";
+
+describe("<Targets />", () => {
+    let props;
+    const state = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['state' is assigned a value but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4894825)